pages/bitcoin-core-extensions/electrs.en.mdx

```diff
@@ -57,31 +223,31 @@ electrum_rpc_addr = "0.0.0.0:50001"
 log_filters = "INFO"
 ```
 
-Sla het op met `Ctrl + X` en bevestig met `Y`. Je kunt je even kijken of het werk met hetzelde commando als daarnet: `./target/release/electrs`. Als het goed is komt er nu een output op je scherm waaruit blijkt dat Electrs gestart is en begonnen is met indexeren.
+Save it with `Ctrl + X` and confirm with `Y`. You can check if it works with the same command as before: `./target/release/electrs`. You should now see an output on your screen showing that Electrs has started indexing.
 
 ## Firewall
 
-Het Electrum protocol communiceert over een vaste set poorten. Je kunt 50001 open zetten als je met bijvoorbeeld Sparrow wallet op je laptop wil praten met je Electrum Server.
+The Electrum protocol communicates over a fixed set of ports. You can open 50001 if you want to talk to your Electrum Server using, for example, Sparrow wallet on your laptop.
 
 ```bash
-sudo ufw allow 50001 comment "Port voor Electrum Server"
+sudo ufw allow 50001 comment "Port for Electrum Server"
 ```
 
-Wil je van afstand Electrs monitoren dien je ook 4224 open te zetten. Dit is optioneel.
+If you want to monitor Electrs remotely you also need to open 4224. This is optional.
 
 ```bash
-sudo ufw allow 4224 comment "Port voor monitoring van Electrs"
+sudo ufw allow 4224 comment "Port for monitoring Electrs"
 ```
 
-## Automatiseren
+## Automate
```

Comment:
Sommige hoofdstukken gebruiken "Automating", sommigen "Automate". Ik zou er 1 kiezen wat betreft consistentie.

Reply:
Automation it is